### PR TITLE
feat: Implement default lf tags for database

### PR DESCRIPTION
**Pull request by svdimchenko**
### Description

Add config to assign default tags for database if it's created by DBT.
Resolves: https://github.com/dbt-athena/dbt-athena/issues/373

## Checklist
- [x] You followed [contributing section](https://github.com/dbt-athena/dbt-athena#contributing)
- [x] You kept your Pull Request small and focused on a single feature or bug fix.
- [ ] You added unit testing when necessary
- [ ] You added functional testing when necessary

### CODE REVIEW

File: dbt/include/athena/macros/adapters/schema.sql [hunk 1]
```diff
@@ -2,6 +2,9 @@
   {%- call statement('create_schema') -%}
     create schema if not exists {{ relation.without_identifier().render_hive() }}
   {% endcall %}
+
+  {{ adapter.add_lf_tags_to_database(relation) }}
+
 {% endmacro %}
```
Comment (nicor88) [inline dbt/include/athena/macros/adapters/schema.sql:9]: does dbt have an on update schema macro?

**Review by nicor88 — APPROVED** [review]
Great work 💯

File: dbt/include/athena/macros/adapters/schema.sql (hunk 1 above)
Comment (svdimchenko) [reply dbt/include/athena/macros/adapters/schema.sql:9]: @nicor88, I was thinking about the approach when we have database tags config on model level. But this is not good idea, as different models can have different configs for same database. That's why I've felt back to default tags for database on dbt profile level.

Comment (nicor88) [reply dbt/include/athena/macros/adapters/schema.sql:9]: ah good point - yeah, then better this way, tags for specific models can be overwritten per model basis - so all good.

**Pull request merged**